test/JDBC/expected/TestXML.out

```diff
@@ -27,3 +27,15 @@ xml
 ~~END~~
 
 DROP TABLE XML_dt;
+
+CREATE TABLE [dbo].[notification_definition]([id] [bigint] NOT NULL,[name_tx] [nvarchar](200) NOT NULL,[description_tx] [nvarchar](max) NULL,[schedule_xml] [xml] NULL,[default_sender_tx] [nvarchar](200) NULL,[lock_id] [tinyint] NOT NULL,[settings_xml] [xml] NULL,[agency_id] [bigint] NULL)
```

Comment:
Have a test to insert NULL through prep-exec also

Reply:
Added tests which insert NULL through prep-exec in the next revision